### ISSUE TITLE
Additional Systems

### ISSUE THREAD

**Issue by ravensvoice** [plaidman/retromega-next#13]
Would love to get system pages/images for Atari 2600, Atari 5200, Atari 7800, Colecovision, Intellivision and Atari Jaguar.

Atari Lynx is also missing an image.

**Comment by plaidman** [comment]
Sorry I don't plan on adding more images unless the original author decides to create some new ones. My photoshop-fu is very weak so creating a new image is really only feasible for me IF I can find a good source image, which often takes hours of scouring the internet, then further hours of touchups. And I usually don't find a good source image.

I am open to wholly replacing all images if someone can suggest a good source with more images, where they are all following a cohesive design and they look good with the theme. I am a stickler for quality and I only want the best for the theme.

**Comment by plaidman** [comment]
I'm going to leave this issue open so people can use it to suggest theme images if they find good ones

**Comment by ravensvoice** [comment]
@plaidman can you make the pages for them, with the years and everything?

Currently I'm using images I pulled from here: https://themizarkshow.com/portfolio/vector-controllers/ for those systems but don't know if you could use them. Thanks!

**Comment by plaidman** [comment]
I noticed those controllers and they look great, sadly he is missing some key systems that would make it just a lateral move if I shifted over to using those images.

I do have an item in my todo list to add more years and colors for the systems I'm missing (I'm going to use emulationstation themes as a resource for which systems to do), I might move that up, even if they don't have images

**Comment by plaidman** [comment]
@ravensvoice I added some of the more popular systems ready to go for the next release. if you want, you can grab [this file](https://github.com/plaidman/retromega-next/blob/main/components/resources/CollectionData.qml) if you want to pull in these changes beforehand.

leaving this issue open for suggestions on system images

**Comment by ravensvoice** [comment]
@plaidman line 103 needs a colon after 'odyssey2' but besides that it works great, thanks! I think the only system that doesn't have a page now (that I have) is colecovision. You are the best. If I come across any controller images will let you know

**Comment by plaidman** [comment]
fixed, thanks!

**Comment by alexb78** [comment]
I personally like this set which looks nice here in this theme as well:
https://www.reddit.com/r/miniSNESmods/comments/79y521/my_updated_custom_folders/
![Screenshot_20220603-090642](https://user-images.githubusercontent.com/53006650/171805917-04f4fefd-72d7-4d15-8ce6-686b8127cf48.png)

**Comment by plaidman** [comment]
@pjankulovski that one is too dark and shiny around the screen area. reflections are super subdued on this theme, if at all.

@alexb78 those folders look awesome, and this is exactly the type of thing I'm looking for, but this particular package is missing some very popular systems like DS and Dreamcast. I may include some extra incomplete packages like these in a separate directory to give some easy access options if people want to switch.

**Comment by plaidman** [comment]
@alexb78 I looked through the guy's post history and it turns out he updated his folder pack and added images for more systems. I'll probably go ahead and bring in his changes!

**Comment by alexb78** [comment]
> @alexb78 I looked through the guy's post history and it turns out he updated his folder pack and added images for more systems. I'll probably go ahead and bring in his changes!

Yeah.. https://www.reddit.com/r/miniSNESmods/comments/9a28i7/custom_folders_update/

**Comment by plaidman** [comment]
my next update will include image packs that include more images (up to 70 systems in one of the packs) and metadata for each.

**Comment by alexb78** [comment]
I have added another set to my repo I liked having it in Arc Browser. Its the well known set from "Ruckage" which is more pixel style. Maybe needs some resizing because the icons become a bit blurry without adjustments. Can provide it as a pull request as well.

![gba-transparent](https://user-images.githubusercontent.com/53006650/172713276-a83d432b-2db8-4349-aa50-3ad8e1fa2441.png)

**Comment by plaidman** [comment]
@alexb78 check [DEVICES.md](DEVICES.md), specifically at the bottom where it's talking about CollectionItem.qml. there's a setting to adjust smoothing when resizing you can turn it off to make pixel themes look better. I'll see if I can find that theme package and maybe include it in a future update.

**Comment by Zenijin** [comment]
I replaced the RP2 image on the Android collection with an AYN Odin picture I did up, since that's the device I'm using.
Would be cool to have it included with the theme and swappable via UI option, but Odin users are probably a minority of people using this theme.

![android](https://user-images.githubusercontent.com/10935361/185980466-90724951-9bf1-4c6b-b92e-a2555dae7141.png)
